src/backend/utils/formatHelper.ts

```diff
@@ -1,5 +1,5 @@
 export function formatBinary(value: number): string {
-    if (value === undefined || value === null) {
+    if (value === undefined || value === null || (value as any) === false) {
```

Comment:
This is one of the weird cases I called out in the initial PR; it's only here because in _some_ case when capturing from WebXR (using the basic immersive web samples), we end up with a `formatBinary` call where the value is `false`, and this broke otherwise.

I'll need to investigate deeper as to why that happens.

Reply:
So this appears to be a problem with the samples on `immersive-web/samples` (which I was using for testing), specifically this line: https://github.com/immersive-web/webxr-samples/blob/ec08873158b74f5a02ecdd5ba250c7ab9ca0fc83/js/render/core/renderer.js#L988

There, the code is calling `gl.stencilMask` with a boolean argument (true / false), but Spector was expecting a number, and so we end up going down the formatBinary code path with a false value.

Either way, it looks like we shouldn't be going down the `formatBinary` code path with a boolean, so I can revert this check.